src/main/java/com/lorepo/icplayer/client/PlayerEntryPoint.java

```diff
@@ -485,4 +486,38 @@ public String getMathJaxRendererOption() {
 
 		return "";
 	}
+
+	public static native void sendEventOnEnd () /*-{
+		var totalTime = 0;
+		
+		var sendEventInterval = setInterval(function () {
+			var minLoadingTime = 500;
+			var isQueueEmpty = $wnd.MathJax.Hub.queue.queue.length === 0;
+			var isIconVisible = @com.lorepo.icplayer.client.PlayerEntryPoint::isLoadingIconVisible()();
+			var isImageLoaded = @com.lorepo.icplayer.client.PlayerEntryPoint::isImageLoaded()();
+			var areAddonsLoaded = isQueueEmpty && !isIconVisible && isImageLoaded;
+
+			if ((areAddonsLoaded && totalTime > minLoadingTime) || totalTime > 5000) {
+				@com.lorepo.icplayer.client.PlayerEntryPoint::sendAddonsLoadedEvent()();
+				clearInterval(sendEventInterval);
+			}
+			totalTime += 200;
+		}, 200);
+	}-*/;
+
+	public static native boolean isLoadingIconVisible () /*-{
+		var element$ = $wnd.$(".image-viewer-loading-image");
+
+		return element$.get(0) ? element$.css('display') === 'block' : false;
+	}-*/;
+
+	public static native boolean isImageLoaded () /*-{
+		var element$ = $wnd.$(".addon_Image_Identification");
+
+		return element$.get(0) ? element$.children('div').length > 0 : true;
+	}-*/;
+
+	public static native void sendAddonsLoadedEvent () /*-{
+		$wnd.parent.postMessage('ADDONS_LOADED', '*');
```

Comment:
Coś nie działa i ciężko mi stwierdzić co za to odpowiada. 
Po wywołaniu player.onPageLoaded(()=> console.log("Page loaded")) nie przychodzi event ADDONS_LOADED
![image](https://github.com/user-attachments/assets/5a2715a4-be8d-459b-a4da-8320c4f8271a)